Dockerfile

```diff
@@ -7,7 +7,7 @@ MAINTAINER Dan Liew <daniel.liew@imperial.ac.uk>
 
 ENV LLVM_VERSION=3.4 \
     SOLVERS=STP:Z3 \
-    STP_VERSION=master \
+    STP_VERSION=2.1.0 \
```

Comment:
@ccadar please use version 2.1.2 as recommended both on STP github and in our documentation